app/reducers/settings.js

```diff
@@ -15,6 +17,7 @@ const initialState = {
     autodetected: true,
     path: null,
     memory: 3096,
+    overrideMemory: 3096,
```

Comment:
If I change the override memory for one instance, it changes it for all the instances